service-commands/scripts/checkAllCid.js

```diff
@@ -111,22 +122,26 @@ async function getClockValues (creatorNode, walletPublicKeys) {
  */
 async function getCidsExist (creatorNode, cids, batchSize = 500) {
   try {
-    const supported =
-      (
-        await makeRequest({
-          method: 'post',
-          url: '/batch_cids_exist',
-          baseURL: creatorNode,
-          data: { cids: [] },
-          validateStatus: (status) => status === 200 || status === 404
-        })
-      ).status === 200
+    if (cidsExistsSupported[creatorNode] === undefined) {
+      cidsExistsSupported[creatorNode] =
+        (
+          await makeRequest({
+            method: 'post',
+            url: '/batch_cids_exist',
+            baseURL: creatorNode,
+            data: { cids: [] },
+            validateStatus: status => status === 200 || status === 404
+          })
+        ).status === 200
 
-    if (!supported) {
-      throw new Error('creator node out of date')
+      if (cidsExistsSupported[creatorNode]) {
```

Comment:
why is this await necessary?

Reply:
If the request succeeds we sleep to avoid rate limit errors, I noticed that these numbers worked slightly better than before.